### ISSUE TITLE
Emoji picker plugin displays emojis "black on white" in Xubuntu 18.04.1 LTS and Chrome Version 70.0.3538.67 (Official Build) (64-bit)

### ISSUE THREAD

**Issue by TheMythPT** [kiwiirc/kiwiirc#615]
I've installed the emoji picker plugin and currently it displays the emojis all in "black on white" format as it can be seen [here](http://i.imgur.com/GnUBO3h.png) while in FireFox Quantum 62.0.3 (64 bits) some of them are displayed graphically, like it can be seen [here](http://i.imgur.com/RehYWMn.png)

**Comment by TheMythPT** [comment]
So, after a few minutes of research and tries, I've found a solution to my problem as it can be seen [here](http://i.imgur.com/nQ8mbu0.png).
I've found [this page](https://www.maketecheasier.com/use-emojis-in-linux/) and after I've done `sudo apt-get install ttf-ancient-fonts`, Chrome now displays the emojis of the emoji plugin in native sheet mode.
Hope this can help someone in the future.

P.S. - I'm currently keeping this issue open to see if there is another possible workaround at a plugin level. Otherwise, feel free to close it.

Cheers
